certified-connectors/Talkdesk/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3046 @@
+                            "description": "A boolean whether the mood settings are enabled or not.",
+                            "title": "Enabled",
+                            "x-ms-visibility": "advanced",
+                            "enum": [
+                              true,
+                              false
+                            ],
+                            "default": false
+                          }
+                        },
+                        "description": "Mood settings for the given agent.",
+                        "title": "Mood settings"
+                      },
+                      "outbound_caller_id": {
+                        "type": "string",
+                        "description": "Outbound caller id for the agent. (>= 5 characters,<= 40 characters)",
+                        "title": "outbound_caller_id",
+                        "x-ms-visibility": "advanced"
+                      },
+                      "outbound_caller_id_selection_enabled": {
+                        "type": "boolean",
+                        "description": "A boolean whether the outbound caller id selection is enabled or not.",
+                        "title": "Outbound caller id selection enabled",
+                        "x-ms-visibility": "advanced",
+                        "enum": [
+                          true,
+                          false
+                        ],
+                        "default": false
+                      },
+                      "phone_number": {
+                        "type": "string",
+                        "description": "The phone number associated to the agent. (>= 5 characters,<= 40 characters)",
+                        "title": "Phone number",
+                        "x-ms-visibility": "advanced"
+                      },
+                      "role_name": {
+                        "type": "string",
+                        "description": "Agent's role.",
+                        "title": "Role name",
+                        "x-ms-visibility": "important",
+                        "enum": [
+                          "Administrator",
+                          "Agent",
+                          "Supervisor"
+                        ]
+                      }
+                    },
+                    "required": [
+                      "email",
+                      "full_name",
+                      "role_name"
+                    ]
+                  },
+                  "description": "An array of agent data to be used to create the agents.",
+                  "title": "Agents"
+                }
+              },
+              "required": [
+                "agents"
+              ]
+            }
+          }
+        ],
+        "x-ms-visibility": "advanced"
+      }
+    },
+    "/calls/callback": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "Success.",
+            "schema": {}
```

Comment:
@StefanCRT22 Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger. **This applies to all the empty schemas in the swagger.**